src/database/queries/userExtension.js

```diff
@@ -344,4 +344,23 @@ module.exports = class MenteeExtensionQueries {
 			throw error
 		}
 	}
+	static async findOneFromView(userId) {
+		try {
+			let query = `
+				SELECT *
+				FROM ${common.materializedViewsPrefix + MenteeExtension.tableName}
+				WHERE user_id = :userId
```

Comment:
@nevil-mathew add get only required fields

Reply:
We cannot add projection here because, if there's any new entity type is added, that field will be missed out